waybackpy/exceptions.py

```diff
@@ -23,44 +22,39 @@ class RedirectSaveError(WaybackError):
     redirect URL is archived but not the original URL.
     """
 
-    pass
-
```

Comment:
Isn't the pass keyword required in Classes that are empty?  Wouldn't it and other such changes in the PR raise a SyntaxError?

Reply:
Okay, non-issue as per <https://stackoverflow.com/q/49754137>.